commands.py

```diff
@@ -278,3 +278,99 @@ def run_discord_bot():
     click.echo('Running bot')
 
     rwrs_discord_bot.run()
+
+
+@app.cli.command()
+@click.argument('database', type=click.Choice(rwr.constants.VALID_DATABASES))
```

Comment:
Is this argument really useful? Why not looping through the valid databases inside the command?